### DISCUSSION TITLE
[Feature] auto-sharing detected face

### DISCUSSION THREAD

**Discussion by zerion87** [immich-app/immich#4936] [Feature Request]
### The feature

Like another solution, i wish that it will be possible to share and integrade the pictures in the libary of the other person, pictures with faces of special persons like kids.

Person A shares all pictures with the face of Person B to Person B. Person B will get a copy of this picture in his gallery.
Or Person A shares face of Person C to Person B. For emaple pictures of the Kids.

And not in Albums. Share directly with a copy.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by wljohnson05** [comment]
This is my biggest wish at the moment as well.

**Comment by derpderpington** [comment]
I would greatly appreciate this feature as well as it is the only reason I miss google photos.

**Comment by ds-sebastian** [comment]
This would be fantastic. I could limit the photos shared with other people based on if they are in the photo or not. Sharing the entire library is a bit much and this could help narrow it down

**Comment by bo0tzz** [comment]
This will be possible with #1673
